src/Microsoft.IdentityModel.JsonWebTokens/JwtHeaderParameterNames.cs

```diff
@@ -28,69 +28,69 @@
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
     /// <summary>
-    /// List of header parameter names see: http://tools.ietf.org/html/rfc7519#section-5.
+    /// List of header parameter names see: https://datatracker.ietf.org/doc/html/rfc7519#section-5.
     /// </summary>
     public struct JwtHeaderParameterNames
     {
         /// <summary>
-        /// see:https://tools.ietf.org/html/rfc7515#section-4.1.1
+        /// see:https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1
```

Comment:
nit: See: https, we probably need a space here

Reply:
also other change in this file as well

Reply:
fixed in commit adbc4d9b31ee1f48b70e12644e833dc636aa9089